specs/decisionmatrix/azure.yaml

```diff
@@ -219,222 +219,24 @@ rows:
           region: "*"
           min_size: 32
-          max_size: 64
-          priority: 2
-          thin_provisioning: false
-          drive_type: "UltraSSD_LRS"
-        - min_iops: 240
-          max_iops: 240
-          instance_type: "*"
-          instance_max_drives: 8
-          instance_min_drives: 1
-          region: "*"
-          min_size: 64
-          max_size: 128
-          priority: 2
-          thin_provisioning: false
-          drive_type: "UltraSSD_LRS"
-        - min_iops: 500
-          max_iops: 500
-          instance_type: "*"
-          instance_max_drives: 8
-          instance_min_drives: 1
-          region: "*"
-          min_size: 128
-          max_size: 256
-          priority: 2
-          thin_provisioning: false
-          drive_type: "UltraSSD_LRS"
-        - min_iops: 1100
-          max_iops: 1100
-          instance_type: "*"
-          instance_max_drives: 8
-          instance_min_drives: 1
-          region: "*"
-          min_size: 256
-          max_size: 8192
-          priority: 2
-          thin_provisioning: false
-          drive_type: "UltraSSD_LRS"
-        - min_iops: 2300
-          max_iops: 2300
-          instance_type: "*"
-          instance_max_drives: 8
-          instance_min_drives: 1
-          region: "*"
-          min_size: 512
-          max_size: 8192
-          priority: 2
-          thin_provisioning: false
-          drive_type: "UltraSSD_LRS"
-        - min_iops: 5000
-          max_iops: 5000
-          instance_type: "*"
-          instance_max_drives: 8
-          instance_min_drives: 1
-          region: "*"
-          min_size: 1024
-          max_size: 8192
-          priority: 2
-          thin_provisioning: false
-          drive_type: "UltraSSD_LRS"
-        - min_iops: 7500
-          max_iops: 7500
-          instance_type: "*"
-          instance_max_drives: 8
-          instance_min_drives: 1
-          region: "*"
-          min_size: 2048
-          max_size: 8192
-          priority: 2
-          thin_provisioning: false
-          drive_type: "UltraSSD_LRS"
-        - min_iops: 16000
-          max_iops: 16000
-          instance_type: "*"
-          instance_max_drives: 8
-          instance_min_drives: 1
-          region: "*"
-          min_size: 8192
-          max_size: 16384
-          priority: 2
-          thin_provisioning: false
-          drive_type: "UltraSSD_LRS"
-        - min_iops: 18000
-          max_iops: 18000
-          instance_type: "*"
-          instance_max_drives: 8
-          instance_min_drives: 1
-          region: "*"
-          min_size: 16384
-          max_size: 32768
-          priority: 2
-          thin_provisioning: false
-          drive_type: "UltraSSD_LRS"
-        - min_iops: 20000
-          max_iops: 20000
-          instance_type: "*"
-          instance_max_drives: 8
-          instance_min_drives: 1
-          region: "*"
-          min_size: 32768
           max_size: 32768
```

Comment:
Ultra disk's capacity can technically go up to 64T. It might need additional approval from Azure support, but it can. We should keep out size at 64T as well so we don't block a customer when they have the necessary permission. Azure API will anyway return an error if they don't have that permission.
https://learn.microsoft.com/en-us/azure/virtual-machines/disks-types#ultra-disk-size

Reply:
updated